lib/fog/libvirt/models/compute/templates/server.xml.erb

```diff
@@ -46,7 +46,7 @@
     </console>
     <input type='tablet' bus='usb'/>
     <input type='mouse' bus='ps2'/>
-    <graphics type='<%= display[:type] %>' port='<%= display[:port] %>' autoport='yes' <% if display[:listen] and !(display[:listen].empty?)  %> listen='<%= display[:listen] %>'<% end %> <% if display[:password] and !(display[:password].empty?) %>passwd='<%=display[:password] %>'<% end %> />
+    <graphics type='<%= display[:type] %>' <% if display[:port] and !(display[:port].empty?) %>port='<%= display[:port] %>' autoport='no' <% else %>port='-1' autoport='yes' <%end%> <% if display[:listen] and !(display[:listen].empty?)  %> listen='<%= display[:listen] %>'<% end %> <% if display[:password] and !(display[:password].empty?) %>passwd='<%=display[:password] %>'<% end %> />
```

Comment:
Sure. i mean by just breaking it into more lines. 2 to define the content of the variable and on the original one just print its content. Got it?